src/mozilla_sec_eia/ex_21/rename_labeled_filings.py

```diff
@@ -0,0 +1,34 @@
+"""Rename labeled filings in GCS after importing from Label Studio."""
+
+import json
+import logging
+
+from mozilla_sec_eia.utils.cloud import GCSArchive
+
+logger = logging.getLogger(f"catalystcoop.{__name__}")
+
+
+def rename_filings():
+    """Rename labeled filings in GCS after importing from Label Studio.
+
+    After importing labeled documents from Label Studio into GCS the
+    labeled items in the bucket do not share the same filename of the
+    unlabeled version of that document, instead the items are just
+    named with the task number from Label Studio. This script looks
+    inside the labeled JSON and gets the filename of the unlabeled
+    document. It then labels the labeled item in GCS with that
+    filename.
+    """
+    archive = GCSArchive()
+    bucket = archive._filings_bucket
+
+    labeled_bucket_name = "labeled"
```

Comment:
I moved this script into the `ex_21` directory. Also, do we still need this `labeled_bucket_name` variable, or should the `GCS_LABELS_BUCKET_NAME` environment variable just include `labeled` at the end of the path?

Reply:
oh okay, seems like it's better to just have the top level bucket name as the environment variable, and then specify `labeled` as the glob to get?

Reply:
My idea here is that by pointing to the top level bucket instead of the individual folder, we have access to anything in that bucket if we/want need. It would probably be a pain if we all had our environments pointing to a single folder, then decided we actually wanted to access something from another folder in the bucket and then all have to recreate our envs.